subgraph/packages/subgraph-dufour-channel/schema.graphql

```diff
@@ -0,0 +1,57 @@
+type Account @entity {
+  "account's address"
+  id: ID!
+  "account's public key"
+  publicKey: Bytes
+  multiaddr: [String!]!
+  "channels where the Account is the source"
+  fromChannels: [Channel!]! @derivedFrom(field: "source")
+  "channels where the Account is the destination"
+  toChannels: [Channel!]! @derivedFrom(field: "destination")
+  "number of outgoing channels"
+  fromChannelsCount: BigInt!
+  "number of incoming channels"
+  toChannelsCount: BigInt!
+  "has the Account set a multiaddr?"
+  hasAnnounced: Boolean!
+  "sum of the channel balances where the account is the source"
+  balance: BigDecimal!
+}
+
+enum ChannelStatus {
+  CLOSED
+  OPEN
+  PENDING_TO_CLOSE
+}
+
+type Ticket @entity {
```

Comment:
One field is missing: "indexOffset". Each ticket redemption can redeem a group of tickets. The size of the group is the `indexOffset`. This value can be found as parameters in the ticket redemption txn

Reply:
indexOffset is not in the ticket redemption txn, but in the ticket redeem call.. Or I cannot find it.